brightray/browser/notification_presenter.h

```diff
@@ -22,6 +22,7 @@ class NotificationPresenter {
 
   base::WeakPtr<Notification> CreateNotification(
       NotificationDelegate* delegate);
+  virtual void RemoveAllNotifications();
```

Comment:
_(minor)_ Notifications seems implicit, could this be just `RemoveAll` -- or, for consistency with the new atom code, `CloseAll`?